examples/ref-templates/assets/dynamic-fee.py

```diff
@@ -0,0 +1,100 @@
+from pyteal import *
+
+#replace the following params with your custom values
+john = Addr("2UBZKFR6RCZL7R24ZG327VKPTPJUPFM6WTG7PJG2ZJLU234F5RGXFLTAKA")
+master = Addr("JAKNKAMEW63I3H3AXWINXEWW6CCLKX4MQGNTTNBSSUHNCHSLQURMMO47XM")
```

Comment:
Please change this address to master which is in `algob.config.js`

Reply:
ok